test/bpf/Makefile

```diff
@@ -1,6 +1,6 @@
 include ../../Makefile.defs
 
-FLAGS := -I../../bpf/ -I../../bpf/include -I. -D__NR_CPUS__=$(shell nproc) -O2
+FLAGS := -I../../bpf/ -I../../bpf/include -I. -D__NR_CPUS__=$(shell nproc -all) -O2
```

Comment:
This should be `nproc --all`, I think. Currently I see

```
make -C test/bpf/

nproc: invalid option -- 'a'

Try 'nproc --help' for more information.
```

on `v1.7` builds. I don't think this is a big issue since it only affects the BPF tests, but will send a PR with a fix nevertheless.

Reply:
Oh, that seems to have been a typo I made while resolving a conflict. Thanks a lot for the fix!